src/main/java/com/blackducksoftware/integration/fortify/batch/model/VulnerableComponentView.java

```diff
@@ -45,51 +45,38 @@
 
     private VulnerabilityWithRemediationView vulnerabilityWithRemediation;
 
-    public String getComponentName() {
-        return this.componentName;
+    public VulnerableComponentView(String componentName, String componentVersionName, String componentVersionOriginName, String componentVersionOriginId,
+            String componentVersionLink, VulnerabilityWithRemediationView vulnerabilityWithRemediation) {
+        super();
```

Comment:
I don't think we need 'super()' here.

Also, the fields in this class can be marked final right?